tools/config_updater.py

```diff
@@ -593,7 +689,12 @@ def main(unused_argv):
   token = GetToken()
   config_lib.CONFIG.AddContext("Commandline Context")
   config_lib.CONFIG.AddContext("ConfigUpdater Context")
-  startup.Init()
+
+  if flags.FLAGS.subparser_name == "initialize":
+    startup.ConfigInit()
+    Initialize(config_lib.CONFIG, token=token)
```

Comment:
I don't think it matters since everything else is conditional, but it shouldn't hurt to add.  I will put that in and make sure it doesn't cause anything weird and push it up